src/ServiceControl/SetupBootstrapper.cs

```diff
@@ -114,9 +110,37 @@ static TransportSettings MapSettings(Settings settings)
             return transportSettings;
         }
 
-        readonly Settings settings;
+        static void SetTransportSpecificFlags(SettingsHolder settings, string poisonQueue)
```

Comment:
Can/should we move all of these into the transport specific packages? i.e. Couldn't TransportCustomization include a `CreateQueues` and then have a virtual method to update the settings?